src/web/server/htmlTemplate.ts

```diff
@@ -199,6 +201,14 @@ export const htmlTemplate = ({
                     })(window, document);
                 </script>
 
+                <script>
+                    if (window.guardian.mustardCut === false) {
+                        window.guardian.gaPath = "${gaPath.legacy}"
```

Comment:
Maybe you could just move these to within the two script tags that set the mustard cutting? https://github.com/guardian/dotcom-rendering/pull/2091/files#diff-2767f0a6851c0895c23f2fd1528f15726d84dbd13203a3d96247f5dd1e107564R146

Reply:
We thought about that but then we saw another use case [further up](https://github.com/guardian/dotcom-rendering/blob/main/src/web/server/htmlTemplate.ts#L158) where there was a conditional around `mustardCut` like this. That said, I see that this use case is inside a function so yeah, could be moved.

Reply:
Excellent, done